src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -119,10 +123,13 @@ export function useSwapInputsController({
       return addCommasToNumber(inputValues.value.inputAmount, '0');
     }
 
+    const assetBalanceDisplay = internalSelectedInputAsset.value.balance.display ?? '';
```

Comment:
Why the `display` amount? Also should the fallback be 0?

Reply:
This was my attempt to get as close to what we display in the balance badge for selling 100%